admin-frontend/src/utils.js

```diff
@@ -35,3 +35,4 @@ export const ageValidator = (value, allValues) => {
 export const getYouthClubs = () => dataProvider(GET_LIST, 'youthClub')
   .then((res) => res.data.map((youthClub) => ({ id: youthClub.name, name: youthClub.name }))) // TODO: Eventually it will be wise to change the ID to the actual ID. This will require backend changes.)
 
+export const isSubstring = (mainString, subString) => mainString.includes(subString);
```

Comment:
With my previous comment I meant that there's no need for this isSubstring function, just use the string.includes("substring") directly wherever you need it as it's actually a couple of characters shorter even + standard. But fix it in another commit